web-ui/src/main/resources/catalog/components/viewer/baselayerswitcher/BaseLayerSwitcherDirective.js

```diff
@@ -58,6 +58,16 @@
             layers.insertAt(0, layer);
             return false;
           };
+
+          scope.dismissPopover = function() {
+            angular.element(element.find("[data-ngeo-popover]")[0]).scope().popoverCtrl.dismissPopover()
```

Comment:
Don't think this is the way to use it.
Have you tried something like `ngeo-popover-dismiss=".content"` ?

Reply:
Yes, I've tried, but it doesn't work. Ngeo definition for `ngeo-popover-dismiss` is
```javascript
      if (attrs['ngeoPopoverDismiss']) {
        $(attrs['ngeoPopoverDismiss']).on('scroll', function() {
          ngeoPopoverCtrl.dismissPopover();
        });
      }
```
It attaches a listener to the `scroll` event. However, what I'm trying to do here is to close the popup once the user click on one of the available layers. It also closes if the user click outside the list of layers.

Reply:
I've updated the code.